### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by hushui502**

### CODE REVIEW

**Comment by nkysg** [comment]
@hushui502 there are some mistakes in the pull request. please run cmd cargo clippy --all-targets and fix it.

**Comment by nkysg** [comment]
Hi @hushui502 on the starcoin top directory, run ```cargo build``` command, and run ```cd rpc```, then run ```../target/debug/starcoin-rpc-schema-generate```, then run ```git status``` check if some json file need to commit.
Our project is not friendly to new contributor, I will add a check shell script to fix. Thx.

File: vm/stdlib/sources/YieldFarmingV2.move [hunk 1]
```diff
@@ -446,8 +446,8 @@ module YieldFarmingV2 {
                                                    release_per_second: u128): u128 {
         assert!(last_update_timestamp <= now_seconds, Errors::invalid_argument(ERR_FARMING_TIMESTAMP_INVALID));
         let time_period = now_seconds - last_update_timestamp;
-        let addtion_index = release_per_second * ((time_period as u128));
-        harvest_index + mantissa(exp_direct_expand(addtion_index))
+        let addition_index = release_per_second * ((time_period as u128));
```
[the remark this answers is not in the document]
Comment (jolestar) [reply vm/stdlib/sources/YieldFarmingV2.move:449]: @hushui502 can you help to resolve this?

**Comment by jolestar** [comment]
Please help rebase the branch and run scripts/check_commit.sh and commit all generated files. @hushui502

**Comment by nkysg** [comment]
修改代码提交后运行下这个文件 https://github.com/starcoinorg/starcoin/blob/master/scripts/check_commit.sh检查是否有遗漏

**Comment by jolestar** [comment]
I close this PR for a long time no response. Feel free to reopen it or submit a new PR.

**Pull request closed without merging**